src/main/java/be/appwise/core/core/CoreBuilder.kt

```diff
@@ -23,8 +23,12 @@ class CoreBuilder(internal val context: Context) {
 
     /***
      * Initialize this logger
+     *
+     * @param tag The name to be used in LogCat
+     * @param isLoggable Whether or not the logger should be enabled for this application.
+     *      When using in a production/release version it should not be enabled (security reasons) i.e. use BuildConfig.DEBUG
      */
-    fun initializeLogger(tag: String = "AndroidApp", isLoggable: Boolean = BuildConfig.DEBUG): CoreBuilder {
+    fun initializeLogger(tag: String, isLoggable: Boolean): CoreBuilder {
```

Comment:
Wouldn't adding a default value of `false` be safer? So that by default no logs will be sent.

Or at least add in the README to use `BuildConfig.DEBUG` 😉 or both.

Reply:
But... I added a KDoc, already explaining this 😅 

Well, as long as you're not calling this function nothing will be logged with Logger.
By having no default values, the user will be aware of what the 'issue' could be when they don't see any logs appearing.

In any case, I'm still open for suggestions. And I'm updating the ReadMe, and more as we speak, so I can integrate this in there as well.